domain/src/main/java/com/epam/esm/domain/dto/CreateOrderDTO.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.esm.domain.dto;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class CreateOrderDTO {
+
+    @NotNull
+    @Min(value = 1)
+    @Max(value = Long.MAX_VALUE)

```

Comment:
Is it possible to populate this field with value which is greater than Long.MAX_VALUE?

Reply:
actually no, my bad. removed